core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -81,6 +85,7 @@
 
   private ClientStreamListener masterListener;
   private Future<?> scheduledRetry;
+  private double nextBackoffInSeconds;
```

Comment:
I think this name is slightly confusing. It's not the amount of time until the next scheduled attempt (which is `RetryPlan#backoffInMillis`), but rather this defines the interval `(0, nextBackoffInSeconds)` from which the subsequent backoff time can be chosen. Maybe `nextBackoffWindowInSeconds` or `nextBackoffIntervalInSeconds` would be better names?

Reply:
Done.